components/brave_sync/brave_profile_sync_service_impl.cc

```diff
@@ -893,15 +934,35 @@ bool BraveProfileSyncServiceImpl::IsBraveSyncConfigured() const {
          // |OnSetupSyncHaveCode| or in |OnSetupSyncNewToSync|
          (brave_sync_prefs_->GetSyncDevices()->size() >= 2);
 }
+void BraveProfileSyncServiceImpl::FetchDevices() {
+  DCHECK(sync_client_);
+  brave_sync_prefs_->SetLastFetchTime(base::Time::Now());
```

Comment:
If we always `FetchSyncRecords` after `FetchDevices`, there is no meaning keeping this
https://github.com/brave/brave-core/blob/e405f2c3eac7b29d7e036e5b6e98767fe0d17dd7/components/brave_sync/brave_profile_sync_service_impl.cc#L829

Reply:
I removed `SetLastFetchTime` at https://github.com/brave/brave-core/pull/3234/files#diff-ab344835b09fb6ac1c601af787b116bbL788